test/test-functions

```diff
@@ -1032,7 +1032,6 @@ EOF
 DefaultEnvironment=${default_environment[*]}
```

Comment:
commit message: cannot follow the commit hash ba7abf79a5x

Reply:
Whoops, thank you.